rocky/reports/report_types/aggregate_organisation_report/recommendations.html

```diff
@@ -3,9 +3,7 @@
 <section id="recommendations">
     <div>
         <h2>{% translate "Recommendations" %}</h2>
-        {% if total_findings == 0 %}
-            <p>{% translate "There are no recommendations." %}</p>
-        {% else %}
+        {% if total_findings != 0 %}
```

Comment:
`!= 0` might not be needed as an non empty set/list/ not 0 int evaluates to true